src/core/JsonConfigCommandCenter.ts

```diff
@@ -14,18 +18,115 @@ export class JsonConfigCommandCenter{
     }
 
     public static get editor(): CogniteJsonEditor | null {
-        if(JsonConfigCommandCenter.editorInstance) {
+        if (JsonConfigCommandCenter.editorInstance) {
             return JsonConfigCommandCenter.editorInstance;
         } else {
             console.error("Cannot retrieve editor before instantiation!");
             return null;
         }
     }
 
+    public static get currentJson(): any {
+        const currentJsonText = JsonConfigCommandCenter.editor?.getText();
+        if (currentJsonText) {
+            return JSON.parse(currentJsonText);
+        }
+    }
+
+    public static loadDigitalTwins = (digitalTwinConfigMap: (twinNames: Map<number, unknown> | null) => void): void => {
```

Comment:
this function is not very reusable. make this an async function that returns a promise. handle that promise in the component. Since we may want to show errors in the component if any error occurs. Callbacks are old way of writing javascript code. now since async await is available we can use them.